src/sensors/heading/gyroscope/GY50.cpp

```diff
@@ -31,10 +31,8 @@ GY50::GY50(Runtime& runtime, int offset, unsigned long samplingInterval)
 int GY50::getHeading()
 {
     // Get the reading from (-180,180) to [0, 360) scale
-    static constexpr auto kFullScaleDegrees = 360;
-    auto normalizedReading = static_cast<int>(mAngularDisplacement) % kFullScaleDegrees;
-
-    return normalizedReading < 0 ? normalizedReading + kFullScaleDegrees : normalizedReading;
+    auto normalizedReading = static_cast<int>(mAngularDisplacement) % 360;
+    return normalizedReading < 0 ? normalizedReading + 360 : normalizedReading;
```

Comment:
:warning: **readability-magic-numbers** :warning:
360 is a magic number; consider replacing it with a named constant